models/todo.js

```diff
@@ -0,0 +1,16 @@
+var Sequelize = require('sequelize');
+
+module.exports=function(sequelize, DataTypes){ 
+  return Todo = sequelize.define('todoentries', {
+    title: {
+        type: Sequelize.STRING,
+    },
+    completed: { type: Sequelize.BOOLEAN }
```

Comment:
You've made these two definitions inconsistent in your spacing. Pick one and stick to it :)